### PR TITLE
Missing space

### LINKED ISSUE #1: Missing space: "ℹ info Fetching allissues with title "
Coming from https://github.com/JasonEtco/create-an-issue/pull/118#issuecomment-985758788

### PR DESCRIPTION
**Pull request by PF4Public**
I'm assuming, that `searchExistingType` is always defined.

Fixes #1

### CODE REVIEW

**Comment by dblock** [comment]
Sorry I'm late here. Let's add a line to https://github.com/dblock/create-a-github-issue/blob/main/CHANGELOG.md even though this is trivial so it's easier to track what changed between releases?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ### 3.1 (Next)
 
-* Your contribution here.
+* Inserted missing space between `${searchExistingType}` and `issues`.
```
Comment (dblock) [inline CHANGELOG.md:3]: Make it look like the other entries below with the GH issue link? 

I’ll add danger-changelog to get this automated …

**Comment by PF4Public** [comment]
It is indeed trivial, but of you insist :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ### 3.1 (Next)
 
-* Your contribution here.
```
Comment (dblock) [inline CHANGELOG.md:3]: Put this one back.

**Comment by dblock** [comment]
FYI this was released in v3.1.0.

**Pull request merged**